pkg/operator/controller/scc.go

```diff
@@ -39,7 +40,7 @@ import (
 const sccName = "containerized-data-importer"
 
 func setSCC(scc *secv1.SecurityContextConstraints) {
-	scc.Priority = &[]int32{10}[0]
+	scc.Priority = pointer.Int32(0)
```

Comment:
I'm not sure if the best is to set it to 0 explicitely or if it should be left unset.

Reply:
Same result according to the API

Reply:
I would leave it unset, just to be consistent with all the other operators in our eco system.